examples/int8_training/peft_adalora_whisper_large_training.py

```diff
@@ -538,7 +550,16 @@ def main():
     metric = evaluate.load("wer")
 
     # model
-    model = WhisperForConditionalGeneration.from_pretrained(args.model_name_or_path, load_in_8bit=True)
+    if (args.multi_gpu):
```

Comment:
This code is taken from https://github.com/huggingface/peft/issues/242#issuecomment-1491447956